### DISCUSSION TITLE
Getting Access to Server Denied - Previously Working

### DISCUSSION THREAD

**Discussion by gandgpatton** [weather-gov/api#649] [Q&A]
I am getting an access denied error trying to retrieve a forecast.

Access Denied
You don't have permission to access "http://api.weather.gov/gridpoints/PHI/38,113/forecast" on this server.
Reference #18.46e2c17.1690479503.1be11b99

I started getting this error in the beginning of July.  (after running properly for over a year) I'm using node-red to query and parse the forecast every hour so I don't think I'm bumping up against any rate limits.  Also, I can type the above into a web browser and return data.    I am using a user-agent.

Any ideas?

Thanks,
Greg

**Comment by JJKraw** [comment]
Is it possible to post the whole request? Perhaps that would reveal a clue.

**Comment by dwhitemv25** [comment]
You need to use`https`, not `http`. Your web browser is silently redirecting to the `https://` URL.
